### PR TITLE
Bug 1163501 - [Stingray] Dashboard app folder and template

### PR DESCRIPTION
**Pull request by begeeben**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d8a1933fb8827bf7d6757cfe5bd1ba1ff14aeb51

File: tv_apps/dashboard/test/marionette/dashboard_test.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        height: 1080
+      }
+    }
+  };
+
+  var client = marionette.client(opts);
+
+  setup(function() {
+    // Launch test app
+    client.apps.launch(APP_URL);
```
Comment (huchengtw-moz) [inline tv_apps/dashboard/test/marionette/dashboard_test.js]: No need this.

File: tv_apps/dashboard/js/dashboard.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      this.keyNavigationAdapter.on('move', this.onMove.bind(this));
+      this.keyNavigationAdapter.init(this.mainSection);
+    },
+
+    onMove: function (key) {
+      var activeDirection = this.mainSection.dataset.activeDirection;
+
+      switch (activeDirection) {
+        case 'up':
+          if (key === 'down') {
```
Comment (rexboy7) [inline tv_apps/dashboard/js/dashboard.js]: Although it makes sense to me that pressing the opposite direction to move back but the UX spec said we should do that by pressing the key again or pressing back key. May you confirm this with UX team?

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=40f591ca667736b2a640097051c87dc86c364c9a

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4bae1454695bfd65f70cae38ce24d5a5bb3e637e

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ed49496ffe7fdc77bf4b9b3863ce2d734a3d96b4

**Pull request merged**